src/devices/bus/qbus/qbus.cpp

```diff
@@ -16,13 +16,15 @@
 #include "qtx.h"
 #include "dvk_kgd.h"
 
+#include "dvk_mx.h"
 
 void qbus_cards(device_slot_interface &device)
```

Comment:
The new `#include` should be grouped with the other peripheral board headers, with the two blank lines separating them from the function.